libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/EHorizonMapper.kt

```diff
@@ -70,6 +70,7 @@ internal object EHorizonMapper {
             speedLimit,
             laneCount,
             meanElevation,
+            countryCodeIso2,
```

Comment:
@mskurydin @SiarheiFedartsou could you also rename `countryCode` -> `countryCodeIso3` on NN side